public/js/components/stories/Frames.js

```diff
@@ -1,46 +1,26 @@
 "use strict";
 
 const React = require("react");
-const { DOM: dom, createElement } = React;
-const { Provider } = require("react-redux");
-const { Map, fromJS } = require("immutable");
+const { DOM: dom, createElement, createFactory } = React;
+const { renderComponent, storiesOf } = require("./utils");
 
-const { createStore } = require("./utils");
+const Frames = require("../Frames");
 
-const { storiesOf } = require("@kadira/storybook");
-const Frames = React.createFactory(require("../Frames"));
-const fixtures = require("../../test/fixtures");
+const style = {
+  width: "300px",
+  margin: "auto",
+  paddingTop: "100px" };
 
-function getData(fixtureName) {
-  const fixture = fixtures[fixtureName];
-  const frames = fixture.pause.frames;
-  return {
-    pause: Map({ frames }),
-    sources: fromJS({ sources: fixture.sources.sources })
-  };
+const component = dom.div(
+  { className: "accordion" },
+  dom.div({ className: "_content" }, createElement(createFactory(Frames)))
+);
+
+function renderFrames(fixtureName) {
+  return renderComponent(component, fixtureName, { style })
 }
 
 storiesOf("Frames", module)
-  .add("Blank", () => {
-    return renderContainer(Frames);
-  })
-  .add("TodoMVC Todo Toggle", () => {
-    return renderContainer(Frames, getData("todomvcToggle"));
-  })
-  .add("Nested Closures", () => {
-    return renderContainer(Frames, getData("pythagorean"));
-  });
-
-function renderContainer(Component, data) {
-  const store = createStore(data);
-  return dom.div(
-    { style: {
-      width: "400px",
-      margin: "auto",
-      paddingTop: "100px"
-    }},
-    createElement(Provider, { store },
-      dom.div({ className: "accordion" },
-        dom.div({ className: "_content" }, createElement(Component))))
-  );
-}
+  .add("Not Paused", () => renderFrames("todomvc"))
+  .add("Event Handler", () => renderFrames("todomvcUpdateOnEnter"))
+  .add("Nested Closures", () => renderFrames("pythagorean"));
```

Comment:
haha - yea, i had the same thought in richmond... but i'm now on the fence. We've discussed how we want to use connect with top-level debugger components (editor, sources, ...). 

I'm leaning towards using connect to pull in as many props as possible here. and then avoiding connect for lower-level components (breakpoint, tree, ...). This would line up nicely with storybook as well, where if we have a story for an Editor it can be rendered solely with the fixture data, wheras a Tree or Accordion story uses props...

I could be persuaded though